src/users/users.router.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const usersRouter = express.Router();
+const { usersController } = require('./users.controller');
+
+usersRouter.get('/users', usersController.getAllUsers);
+
+usersRouter.post('/users', usersController.addNextUser);
+
+usersRouter.delete('/users/:userId', usersController.deleteUser);
+
+usersRouter.get('/users/:userId', usersController.getOneUser);
+
+usersRouter.put('/users/:userId', usersController.userToUpdate);
```

Comment:
Please checkout my suggestions in `src/createServer.js`

```suggestion
usersRouter.get('/', usersController.getAllUsers);

usersRouter.post('/', usersController.addNextUser);

usersRouter.delete('/:userId', usersController.deleteUser);

usersRouter.get('/:userId', usersController.getOneUser);

usersRouter.put('/:userId', usersController.userToUpdate);
```